content/_data/upgrades/2-361-1.adoc

```diff
@@ -0,0 +1,63 @@
+Enthusiastic users may wish to install Java from Adoptium or another vendor.
+Adoptium recently began providing Linux installation packages, as described in link:https://blog.adoptium.net/2021/12/eclipse-temurin-linux-installers-available/[a piece by George Adams].
+The choice of which Java vendor to use is your own, as long as that vendor provides Java 11 or Java 17.
+Refer to your chosen Java vendor for installation instructions.
+
+Once you have installed a suitable version of Java, configuring Jenkins to use that Java runtime is easy.
+The most straightforward way is to configure that version of Java as the default version of Java at the operating system (OS) level:
+
+Debian: `update-alternatives --config java`
+Red Hat: `alternatives --config java`
+SUSE: `update-alternatives --config java`
+
+Alternatively, users who do not wish to change the default version of Java can customize the `JAVA_HOME` or `JENKINS_JAVA_CMD` environment variable as part of the Jenkins `systemd(1)` service unit.
+Refer to the link:https://www.jenkins.io/doc/book/system-administration/systemd-services/[Managing systemd services] section of the Jenkins documentation for more information.
+
+==== Updated Jetty version
+
+The Jetty version has been updated from 9.4.46.v20220331 to 10.0.11.
+Support for OpenSSL-style PEM-encoded RSA private keys has been removed when running Jenkins with the embedded Jetty (Winstone) container and TLS.
+Specifically, the `--httpsPrivateKey` and `--httpsCertificate` flags have been removed in favor of the `--httpsKeyStore` flag.
+The removed flags have had printed deprecation warnings since 2016, and were implemented with non-standard APIs that have since been removed from Java 17.
+The recommendation is to migrate to the `--httpsKeyStore` flag, which takes a keystore as described in link:https://www.jenkins.io/doc/book/installing/initial-settings/#https-with-an-existing-certificate[the documentation].
+As of link:https://openjdk.org/jeps/229[JEP 229], link:https://en.wikipedia.org/wiki/PKCS_12[PKCS12] is the recommended keystore type.
+
+==== Updated minimum supported Remoting version
+
+The minimum required Remoting version has been increased to 4.2.1. Ensure that all agents are running a recent version of Remoting prior to upgrading.
+
+==== Converted `instance-identity` to a separate plugin
+
+Since `instance-identity` is normally a basic part of Jenkins core, this new plugin will be automatically installed.
+If you use an “as-code” installation mechanism, for example a text file with a list of plugins, you may need to add `instance-identity` version 3.1 or later, in order to restore this functionality.
+In particular, inbound agents using TCP (but not WebSocket) transport require this plugin to be installed.
+There may be other cases, which can be identified by use of APIs in the `jenkins.model.identity` package rather than directly accessing `org.jenkinsci.main.modules.instance_identity`.
```

Comment:
Unfortunately [JENKINS-69361](https://issues.jenkins.io/browse/JENKINS-69361) is likely to bite users here. While this should really be fixed prior to the release, if nobody steps up to do that then the least we can do is warn users to make sure `instance-identity` is upgraded to the latest version after upgrading Jenkins. In other words the usual mantra of upgrading your plugins before and after upgrading Jenkins core.

Reply:
Thank you for the insight on that Basil, I appreciate it! I'll be sure to add in that information to make it clear users must be sure to upgrade things accordingly.